File: Languages/C#/basics.cs
using System;

class Program
{
    const int CANT_ELEMS = 2;

    static int Sum(int x, int y)
    {
        return x + y;
    }

    static void Main()
    {
        int[] numbers = new int[CANT_ELEMS];
        int answer = 0;

        // Input loop
        for (int i = 0; i < CANT_ELEMS; ++i)
        {
            Console.Write($"Enter element {i + 1}: ");
            numbers[i] = Convert.ToInt32(Console.ReadLine());
            answer = Sum(answer, numbers[i]);
        }

        // Output
        Console.WriteLine($"Sum: {answer}");
    }
}

File: Languages/C++/basics.cpp
#include <iostream>
using namespace std;

#define CANT_ELEMS 2

int sum(int x, int y) {
    return x + y;
}

int main() {
    int numbers[CANT_ELEMS];
    int answer = 0;

    // Input loop
    for (int i = 0; i < CANT_ELEMS; ++i) {
        cout << "Enter element " << i + 1 << ": ";
        cin >> numbers[i];
        answer = sum(answer, numbers[i]);
    }

    // Output
    cout << "Sum: " << answer << endl;

    return 0;
}